doc/dev/index.rst

```diff
@@ -1119,6 +1119,53 @@ there::
 This will keep the teuthology machine, the logs and the packages-repository
 instance but nuke everything else.
 
+Deploy a cluster for manual testing
+-----------------------------------
+
+The `teuthology framework`_ and `ceph-workbench ceph-qa-suite`_ are 
+versatile tools that automatically provision Ceph clusters in the cloud and
+run various tests on them in an automated fashion. This enables a single
+engineer, in a matter of hours, to perform thousands of tests that would
+keep dozens of human testers occupied for days or weeks if conducted
+manually.
+
+However, there are times when the automated tests do not cover a particular
+scenario and manual testing is desired. It turns out that it is simple to
+adapt a test to stop and wait after the Ceph installation phase, and the
+engineer can then easily ssh into the cluster and do whatever.
+
+This common use case is currently provided for by the following command::
+
+   ceph-workbench ceph-qa-suite --simultaneous-jobs 9 --verbose
+   --teuthology-git-url http://github.com/dachary/teuthology
+   --teuthology-branch openstack --ceph-qa-suite-git-url
+   http://github.com/dachary/ceph-qa-suite --suite-branch wip-ceph-disk
+   --ceph-git-url http://github.com/ceph/ceph --ceph jewel --suite
+   ceph-disk --filter ubuntu_14
+
+This builds packages from the Ceph git repository and branch specified in
+the ``--ceph-git-url`` and ``--ceph`` options, respectively, provisions VMs
+in OpenStack, installs the packages, and deploys a Ceph cluster on them.
+Then, instead of running automated tests, it stops and enters a wait loop.
+
+At this point, the engineer (user) can ssh into the cluster and do whatever
+manual testing is required. This is done by first sshing into the
+teuthology node, as described above, and then sshing to one of the cluster
+nodes. The IP addresses of the cluster nodes can be found in the OpenStack 
+Horizon dashboard. The basic VMs (or "instances" in OpenStack terminology)
+created by `ceph-workbench ceph-qa-suite`_ are named as follows:
+
+``teuthology`` - the teuthology node
+
+``packages-repository`` - VM where packages are stored
+
+``ceph-rpm-...` - VM where packages are built
+
+Any others are the cluster nodes.
```

Comment:
@dachary I'm looking at the teuthology.log of http://149.202.190.143:8081/ubuntu-2016-03-20_10:38:44-ceph-disk-wip-14984---basic-openstack/ (the test mentioned in this section) and I cannot find a **targets** section. Do you mean the **roles** and **openstack** sections?